samples/bluetooth/fast_pair/locator_tag/sysbuild/ipc_radio/boards/thingy53_nrf5340_cpunet.conf

```diff
@@ -7,3 +7,9 @@
 # The FEM TX gain must be changed to achieve the 0 dBm TX power on the
 # Bluetooth controller side.
 CONFIG_MPSL_FEM_NRF21540_TX_GAIN_DB=20
+
+# Disable logging
```

Comment:
Can we comment why we disable logging here (I guess it's because we cannot access UART that outputs the logs easily)?